lrpar/Cargo.toml

```diff
@@ -42,6 +42,7 @@ static_assertions.workspace = true
 vob.workspace = true
 syn.workspace = true
 prettyplease.workspace = true
+web-time = "1.1.0"
```

Comment:
In theory we could also make this an optional dependency, and only conditionally depend on it if a feature is enabled.

That duplicates the conditional compilation in the crate, but I get that some people might want to minimize dependencies.

Reply:
I must admit, I was thinking exactly the same. *Personally* I do have a mild preference for only compiling it on wasm targets, because then non-wasm users don't have to think about it as something they need to audit. But I appreciate that's extra work.

Reply:
What I was thinking was slightly different actually than compiling it conditionally per-platform
But using a feature like the following.

```
#[cfg(feature = "webpolyfills")]
use web_time::{Instant, Duration}

#[cfg(not(feature = "webpolyfills"))]
use std::{Instant, Duration};
```

While the name doesn't imply it you could enable webpolyfills on non-web platforms.
Then you could use the same build options to target either web/non-web.

I think we need this kind of feature anyways, just to enable the library from cargo.toml.
At the very least, I don't know about using target specific cfg options that involve dependencies,
that may very well be a thing though.

Reply:
I *think* we can say something like:

```toml
[target.'cfg(target_arch = "wasm32")'.dependencies]
web-time = "0.1.0"
```

and then:

```rust
#[cfg(target_arch = "wasm32")]
use web_time::{...};
```

But I haven't thought about whether that's nicer than a feature detected in `build.rs` or not TBH. I'm easy either way.